api/nakadi-event-bus-api.yaml

```diff
@@ -925,6 +1173,67 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: |
+      Subscription is a high level consumption unit. Subscriptions allow applications to easily scale the
+      number of clients by managing consumed event offsets and distributing load between instances.
+      The key properties that identify subscription are 'owning_application', 'event_types' and 'consumer_group'.
+      It's not possible to have two different subscriptions with these properties being the same.
+    properties:
+      id:
+        type: string
+        format: uuid
+        readOnly: true
+        description: |
+          Id of subscription that was created. Is generated by Nakadi, should not be specified when creating
+          subscription.
+      owning_application:
+        type: string
+        example: 'gizig'
+        description: |
+          The id of application owning the subscription. The reading and modification of subscription will be limited
+          to this application id.
```

Comment:
@tlossen @dehora well, we just wanted to have some basic security that will not allow users to mess each others' subscriptions. When a new shiny security model for Nakadi will be designed - I don't know, this is not in our hands. Here we just wanted to have some basic minimal stuff. I don't want to wait for the day when some user will mess up other user subscription and that will create a lot of problems. What are you thoughts on that? How we can provide some basic security here?

@tkrop We actually have a separate `consumer_group` property in subscription to differ subscriptions of the same application (and same event-types)
https://github.com/zalando/nakadi/pull/353/files#diff-4492c6b26ee0a37518728ec1585420efR1233